src/envo/envo-edit.owl

```diff
@@ -5448,7 +5464,7 @@ SubClassOf(<http://purl.obolibrary.org/obo/ENVO_00000054> <http://purl.obolibrar
 
 # Class: <http://purl.obolibrary.org/obo/ENVO_00000055> (saline evaporation pond)
 
-AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> "https://en.wikipedia.org/wiki/Salt_evaporation_ponds"^^xsd:string) <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/ENVO_00000055> "A shallow man-made pond designed to produce salt from sea water. The seawater is fed into large ponds and water is drawn out through natural evaporation which allows the salt to be subsequently harvested."^^xsd:string)
+AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> "https://en.wikipedia.org/wiki/Salt_evaporation_ponds"^^xsd:string) <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/ENVO_00000055> "A pond which is 1) shallow, 2) man-made, and 3) designed to produce salt from seawater through evaporation."@en)
```

Comment:
there are also saline evaporation pond adjacent to salt lakes in continental areas, e.g. great salt lake, or to evaporate saline water used to dissolve subsurface salt deposits e.g. around the Luke salt in the Phoenix area, Arizona.